alacritty/src/event.rs

```diff
@@ -767,7 +767,7 @@ impl<N: Notify + OnResize> Processor<N> {
 
     /// Return `true` if `event_queue` is empty, `false` otherwise.
     #[inline]
-    #[cfg(any(target_os = "macos", windows, not(feature = "wayland")))]
+    #[cfg(any(not(feature = "wayland"), target_os = "macos", windows))]
```

Comment:
I've changed this to move the wayland feature **before** the platform checks. This is for uniformity to make it easier and quicker to read these `cfg` annotations, having a uniform structure everywhere.